### DISCUSSION TITLE
Error output encoding

### DISCUSSION THREAD

**Discussion by huuduongnguyen** [PaddlePaddle/PaddleOCR#13162] [Q&A]
Hello, I have some issue with the output encoding. I'm currently using PaddleOCR to detect some of Vietnamese characters, but the output seemed get the error because of encoding type, how can I solve it? The encoding type for Vietnamese is UTF-8. Thanks a lot. Here is an example of the problem
![Screenshot 2024-06-22 234834](https://github.com/PaddlePaddle/PaddleOCR/assets/146931469/7094e4c8-dde6-4b1f-8609-4022e81d0262)

**Comment by UserWangZz** [comment]
What are the instructions you executed?
Is the correct dictionary file given?
Please also make sure that it is not a problem with font display

**Reply by huuduongnguyen** [reply]
Hello, I executed these following codes:

"""
from paddleocr import PaddleOCR,draw_ocr
import matplotlib.pyplot as plt

ocr = PaddleOCR(use_angle_cls=True, lang='vi', use_gpu = True) # need to run only once to download and load model into memory

img_path = r"C:\Users\nguye\Pictures\Screenshots\Screenshot 2023-05-25 083618.png"
result = ocr.ocr(img_path, cls=True)
for idx in range(len(result)):
    res = result[idx]
    for line in res:
        print(line)
        
from PIL import Image
result = result[0]
image = Image.open(img_path).convert('RGB')
boxes = [line[0] for line in result]
txts = [line[1][0] for line in result]
scores = [line[1][1] for line in result]
im_show = draw_ocr(image, boxes, txts, scores, font_path= r"C:\Users\nguye\PaddleOCR\doc\fonts\times.ttf")
im_show = Image.fromarray(im_show)

plt.figure(figsize=(10, 10))
plt.imshow(im_show)
plt.axis('off')  
plt.show()
"""
